### PR TITLE
Fix ebpf cnx

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-compile-tests
run skydive-functional-hw-tests
run skydive-functional-tests-backend-elasticsearch

File: tests/flow_test.go [hunk 1]
```diff
@@ -2311,7 +2312,7 @@ func testFlowsHashCnx(t *testing.T) *Test {
 			foundAB := false
 			foundBA := false
 			for _, f := range flows {
-				if f.Metric.ABPackets != 1 || f.Metric.BAPackets != 1 {
+				if f.Metric.ABPackets == 0 || f.Metric.BAPackets == 0 {
```
Comment (nplanel) [inline tests/flow_test.go]: we should receive exactly the numbers injected here, so the test should be != 10

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by nplanel** [comment]
why reverting "ebpf : use github.com/newtools/ebpf package" ?

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-functional-hw-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-hw-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-hw-tests

**Pull request merged**